packages/chat-stateful-client/review/chat-stateful-client.api.md

```diff
@@ -52,6 +77,9 @@ export type ChatThreadProperties = {
 // @public
 export const createStatefulChatClient: (args: StatefulChatClientArgs, options?: StatefulChatClientOptions | undefined) => StatefulChatClient;
 
+// @public
+export const isChatErrorTarget: (target: string) => boolean;
```

Comment:
This is unfortunate. If there's an idiomatic way to enumerate the values of a string literal union type at runtime, please let me know. The reason this is exposed is so that the adapter can restrict the operations to only the targets acceptable to the stateful client in `useAdaptedSelector` implementation.

Reply:
Yeah unfortunately literal union types don't exist as an object in the transpiled javascript (purely used for typing), is the downsides of this that we have to maintain an array of the known targets?

Reply:
Essentially.

https://www.npmjs.com/package/runtypes could be used here. But we don't have that dependency yet, 
Anyway, I'll move this off the public API, then we can improve upon the implementation later on.

Reply:
Moved implementation to the composite where it's used, off public API.

Can consider using a generic library to get runtime type reflection later.